applications/luci-app-keepalived/htdocs/luci-static/resources/view/keepalived/url.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+'require view';
+'require form';
+
+return view.extend({
+	render: function() {
+		var m, s, o;
+
+		m = new form.Map('keepalived', _('URLs'));
```

Comment:
Please remove this _('URLs') and move it to the m.section. So this looks consistent with the other pages

Reply:
Updated